vpt/cli/sources.py

```diff
@@ -48,8 +48,7 @@ def list_video_inputs():
         cap = cv2.VideoCapture(index)
         if not cap.read()[0]:
             break
-        else:
-            arr.append(index)
+        arr.append(index)
```

Comment:
Why this?

Reply:
Because there is a `break` in the `if` branch. Pylint recommended it

Reply:
Oh, right